<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>783</width>
    <height>323</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="HawaiButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid grey;
border-radius: 5px;</string>
    </property>
    <property name="text">
     <string>Гаваї</string>
    </property>
   </widget>
   <widget class="QPushButton" name="KyivButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid grey;
border-radius: 5px;</string>
    </property>
    <property name="text">
     <string>Київ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LAButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid grey;
border-radius: 5px;</string>
    </property>
    <property name="text">
     <string>Лос-Анджелес</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="nichne_photo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Нічне фото</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>321</width>
      <height>271</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>30</y>
      <width>241</width>
      <height>211</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="LONButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid grey;
border-radius: 5px;
</string>
    </property>
    <property name="text">
     <string>Лондон</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CarrButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid grey;
border-radius: 5px;</string>
    </property>
    <property name="text">
     <string>Каріби</string>
    </property>
   </widget>
   <widget class="QLabel" name="fon_label">
    <property name="geometry">
     <rect>
      <x>-4</x>
      <y>-8</y>
      <width>791</width>
      <height>311</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>fon_label</zorder>
   <zorder>HawaiButton</zorder>
   <zorder>KyivButton</zorder>
   <zorder>LAButton</zorder>
   <zorder>nichne_photo</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>LONButton</zorder>
   <zorder>CarrButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>783</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
